app/dashboard/views/alias_contact_manager.py

```diff
@@ -231,6 +231,7 @@ def delete_contact(alias: Alias, contact_id: int):
 
 @dashboard_bp.route("/alias_contact_manager/<int:alias_id>/", methods=["GET", "POST"])
 @login_required
+@parallel_limiter.lock(name="contact_creation")
```

Comment:
Do you think is there a way to test this in unit test? We can run a redis container when running test.

Reply:
Added a basic unit test.